src/track/keyutils.h

```diff
@@ -16,8 +16,10 @@ class KeyUtils {
         INVALID = 0,
         CUSTOM = 1,
         OPEN_KEY = 2,
-        LANCELOT = 3,
-        TRADITIONAL = 4,
+        OPEN_KEY_AND_TRADITIONAL = 3,
```

Comment:
The existing literals should keep their integer numbers. Otherwise any existing configuration gets tampered and I'm not sure where those values are stored, e.g. the configuration file or the database. The new enum constants should be assigned 5 and 6.

Reply:
Fixed